src/status_im2/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im2.contexts.wallet.create-account.select-keypair.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im2.contexts.wallet.create-account.select-keypair.style :as style]
+    [utils.address :as utils]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def accounts
+  [{:account-props {:customization-color :turquoise
+                    :size                32
+                    :emoji               "\uD83C\uDFB2"
+                    :type                :default
+                    :name                "Trip to Vegas"
+                    :address             "0x0ah...71a"}
```

Comment:
I believe we have access to this information now? should it not be taken from the first address of the user

Reply:
@J-Son89 some data like `emoji` and `color`  seem not be available yet in `(rf/sub [:profile/wallet-accounts])`

Reply:
oh we're not using that, that is for the old wallet? 
I believe we should be taking from `:wallet/accounts`. 

cc @smohamedjavid @briansztamfater

Reply:
since this structure also needs ` :networks      [{:name :ethereum :short-name "eth"}...`
it might be best to add a new sub - `accounts-with-networks` (or something like that) and add this detail here 👍

Reply:
emoji and color are available except in the default account -
we need to address this! see:
https://github.com/status-im/status-mobile/issues/17822